config/database.php

```diff
@@ -45,7 +45,14 @@
             'collation' => 'utf8mb4_unicode_ci',
             'prefix' => env('DB_PREFIX', ''),
             'strict' => env('DB_STRICT_MODE', false),
+            'sslmode' => env('DB_SSLMODE', 'prefer'),
             'timezone' => env('DB_TIMEZONE', Time::getMySQLTimezoneOffset(env('APP_TIMEZONE', 'UTC'))),
+            'options'   => array(
```

Comment:
This codebase does not use `array()`, you should be using `[]` and there should only be a single space between the key and the arrow.